app/services/discovery_engine/sync/locking.rb

```diff
@@ -2,53 +2,9 @@ module DiscoveryEngine::Sync
   # Mixin providing a mechanism for sync operations (`Put`, `Delete`) to lock documents while they
   # are being operated on so that other workers cannot simultaneously update the same document.
   module Locking
```

Comment:
If this module isn't responsible for doing any locking now I think the name should be changed. Something like `Payload`, or `PayloadVersion` etc.

Reply:
Good shout – I was going to remove it in a follow-up PR anyway but it makes sense to rename it so we don't have it hanging around in a weird state!